etc/profile-m-z/rtv.profile

```diff
@@ -1,6 +1,15 @@
 # Firejail profile for rtv
 # Description: Browse Reddit from your terminal
 # This file is overwritten after every install/update
+
+# You can configure rtv to open different type of links
+# in external applications. Configuration here:
+# https://github.com/michael-lazar/rtv#viewing-media-links
+# This include is meant to facilitate that configuration
```

Comment:
Due to the comment added below this sentence can be dropped.

Reply:
i moved the rest of the comment bellow too. It looks out of place otherwise.